pkg/mev/bloxroute_submit_bundle.go

```diff
@@ -14,69 +14,76 @@ import (
 	"github.com/ethereum/go-ethereum/crypto"
 )
 
-type Builder string
+type BlxrBuilder string
 
 const (
-	BuilderBloxroute    Builder = "bloxroute"
-	BuilderFlashbot     Builder = "flashbots"
-	BuilderBeaverBuild  Builder = "beaverbuild"
-	BuilderRsyncBuilder Builder = "rsync-builder"
-	BuilderAll          Builder = "all"
+	BuilderBloxroute    BlxrBuilder = "bloxroute"
+	BuilderFlashbot     BlxrBuilder = "flashbots"
+	BuilderBeaverBuild  BlxrBuilder = "beaverbuild"
+	BuilderRsyncBuilder BlxrBuilder = "rsync-builder"
+	BuilderAll          BlxrBuilder = "all"
 )
 
-// BloxrouteSubmitBundle https://docs.bloxroute.com/apis/mev-solution/bundle-submission
-func BloxrouteSubmitBundle( // nolint: cyclop
-	ctx context.Context, c *http.Client, auth, endpoint string,
-	param *BLXRSubmitBundleParams, options ...BloxrouteSubmitBundleOption,
-) (SendBundleResponse, error) {
-	var opts blxrSubmitBundleOptions
-	for _, fn := range options {
-		if fn == nil {
-			continue
-		}
-		fn(&opts)
-	}
+type BloxrouteBundleSender struct {
```

Comment:
I think we should name it as `BloxrouteClient`, it can have more methods other than `SendBundle`.